docs/src/main/asciidoc/verifier_contract.adoc

```diff
@@ -1499,6 +1499,8 @@ You can now reference your classes in your DSL, as shown in the following exampl
 include::{samples_url}/producer/src/test/resources/contracts/beer/rest/shouldGrantABeerIfOldEnough.groovy[indent=0]
 ----
 
+IMPORTANT: You can setup the Spring Cloud Contract plugin by setting `convertToYaml` to `true`. That way you will NOT have to add the dependency to the JAR with the extended features.
```

Comment:
set the Spring Cloud Contract plugin up

Reply:
Maybe: "With this set to `true` if you use libraries that introduce custom contract processing and extended features, you will not have to add SCC as a dependency to those JARs." ?